frontend/src/APIClients/CampsAPIClient.ts

```diff
@@ -0,0 +1,18 @@
+import { BEARER_TOKEN } from "../constants/AuthConstants";
+import { Camp } from "../types/CampsTypes";
+import baseAPIClient from "./BaseAPIClient";
+
+const getCampById = async (id: string): Promise<Camp> => {
+  try {
+    const { data } = await baseAPIClient.get(`/camp/${id}`, {
+      headers: { Authorization: BEARER_TOKEN },
+    });
+    return data;
+  } catch (error) {
+    return error as Camp;
```

Comment:
Can we actually call this CampResponse instead so we can stay consistent with our types for all the backend responses/return values?

```suggestion
    return error as CampResponse;
```